### PR TITLE
Set up VxAdmin UI scaffolding for super admins (feature-flag gated)

### PR DESCRIPTION
**Pull request by arsalansufi**
# Overview

Issue link: https://github.com/votingworks/vxsuite/issues/1911 (plus https://github.com/votingworks/vxsuite/issues/1912, https://github.com/votingworks/vxsuite/issues/1913, and https://github.com/votingworks/vxsuite/issues/1923)

This PR sets up VxAdmin UI scaffolding for super admins! Gated behind the new `REACT_APP_VX_ENABLE_VVSG2_AUTH_FLOWS` feature flag, a number of tabs have been moved from the admin UI to the super admin UI.

_Super admin UI_

super-admin

_Admin UI_

admin

_Video of super admin UI_

Note that many screens are just stubs that'll be filled out in subsequent PRs.

https://user-images.githubusercontent.com/12616928/174908805-4b6c8d05-95ad-46f4-8378-a707f14b85f1.mov

# Testing

- [x] Added automated tests
- [x] Tested manually for super admins and admins, with and without the feature flag, with and without election definitions

# Checklist

- [ ] ~I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced~ N/A
- [ ] ~I have added JSDoc comments to any newly introduced exports~ N/A
- [x] I have added a screenshot and/or video to this PR to demo the change
- [x] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

**Review by jonahkagan — APPROVED** [review]

File: frontends/election-manager/src/components/navigation_screen.tsx [hunk 1]
```diff
@@ -49,89 +56,102 @@ export function NavigationScreen({
     currentUserSession,
   } = useContext(AppContext);
   const election = electionDefinition?.election;
-  const currentUser = currentUserSession?.type ?? 'unknown';
+  const currentUserType = currentUserSession?.type ?? 'unknown';
+
+  const [showSettingsModal, setShowSettingsModal] = useState(false);
+  const [showLogsModal, setShowLogsModal] = useState(false);
```
[the remark this answers is not in the document]
Comment (arsalansufi) [reply frontends/election-manager/src/components/navigation_screen.tsx:62]: Hmm really good point - let me do some code + design exploration here

[the remark this answers is not in the document]
Comment (arsalansufi) [reply frontends/election-manager/src/components/navigation_screen.tsx:62]: One relevant piece of context: The decision to place "Settings" and "Logs" in the top-right nav as opposed to tabs was an intentional decision on Beau's part to use the top-right nav for election-agnostic items

Of course, we can still address some of the other idiosyncrasies here

[the remark this answers is not in the document]
Comment (arsalansufi) [reply frontends/election-manager/src/components/navigation_screen.tsx]: https://github.com/votingworks/vxsuite/pull/1966/commits/f3c20ffa323abc41e09aba02883d14e7a19249fc

Shifted Settings and Logs from modals to proper routes/screens - still an open question how to indicate that the Settings screen or Logs screen is active beyond the screen title, e.g. modified button styling for the relevant button, similar to what we do for tab nav buttons, but that can be tackled later

[the remark this answers is not in the document]
Comment (jonahkagan) [reply frontends/election-manager/src/components/navigation_screen.tsx]: Sounds good!

**Pull request merged**